### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by drlaw1558**
This PR fixes some typos in the cube_build documentation.

### CODE REVIEW

**Comment by drlaw1558** [comment]
@jemorrison Putting in a PR to fix some minor doc issues.

File: CHANGES.rst [hunk 1]
```diff
@@ -94,6 +94,8 @@ cube_build
 
 - Remove certain WCS keywords that are irrelevant after ``cube_build``. [#6032]
 
+- Fix some typos in the the arguments documentation. [#6077]
```
Comment (hbushouse) [inline CHANGES.rst:97]: this needs to move up to the top section of the log, since it'll be included in 1.2.1

Comment (drlaw1558) [reply CHANGES.rst]: Whoops, missed that section.  Fixed.

**Review by jemorrison — APPROVED** [review]

**Comment by jemorrison** [comment]
Looks good. Thanks for updating the documentation

**Review by hbushouse — APPROVED** [review]

**Pull request merged**